includes/blast_ui.form_advanced_options.inc

```diff
@@ -48,7 +48,7 @@ function blast_ui_blastn_advanced_options_form(&$form, $form_state) {
     '#title' => t('Max target sequences:'),
     '#options' => _get_max_target('blastn'),
     '#default_value' => $defaults['max_target_seqs'],
-    '#description' => t('Select the maximum number of unique target sequences per query sequence to show results for.'),
+    '#description' => t('Select the maximum number of unique target sequences per query sequence to show results for. Results returned may not be the highest scoring hits. <a href="https://academic.oup.com/bioinformatics/article/35/9/1613/5106166">More Information</a>'),
```

Comment:
The change looks great! I just added target to unsure it opens in a new page similar to the e-value documentation. Once you commit this change, I'll merge the branch -thank you for the contribution!
```suggestion
    '#description' => t('Select the maximum number of unique target sequences per query sequence to show results for. Results returned may not be the highest scoring hits. <a href="https://academic.oup.com/bioinformatics/article/35/9/1613/5106166" target="_blank">More Information</a>'),
```